CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ This change log spiritually adheres to [these guidelines](http://keepachangelog.
 Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Security
 
 ## Unreleased
+
+## [327.2]
+
+## Fixed
+
+* `dx upload f1 f2 --visibility hidden` now correctly marks both files as hidden
```

Comment:
This should be added as the next item, below `Reduce the number..` (since this has not been released yet), and the rest of new additions can be removed. 

Once dx-toolkit is released, this `Changelog` will be updated (the unreleased items will be moved to beta and those from beta, to staging).